packages/components/src/components/Table/HighlightContext.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { createContext, useContext } from 'react';
```

Comment:
I think it's not necessary to have a separate file. I'd do it the same way it's done in TableHeader for such a simple context.